lib/sockets/sub-emitter.js

```diff
@@ -35,7 +35,7 @@ function SubEmitterSocket() {
  * @api private
  */
 
-SubEmitterSocket.prototype.onmessage = function(){
+SubEmitterSocket.prototype.onmessage = function(sock){
```

Comment:
Since this arg is never used could you set it back to empty as it was previously?